### PR TITLE
Added User Authenticator System for Issue #5

### PR DESCRIPTION
**Pull request by bvks2020sameer**
Hi according to the open issue of this project I have added the user authentication system file

### CODE REVIEW

**Comment by Anish-M-code** [comment]
@bvks2020sameer the code looks good but it be be better if this code could could be integrated with existing leakylan code and i guess we dont need email and name , just an alphanumeric id and password. The password should be generated by python and displayed in commandline in server side. the user has to manually share the password with person who wants to send / recieve files.

**Comment by bvks2020sameer** [comment]
Hi have made the changes

File: User_Authenticator.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        else :
+            print('User Authentication Sucessfull')
+            self.cursor.close()
+            self.mydb.close()
+            return 1
+        
+    
+
+
+host = 'localhost'
```
Comment (Anish-M-code) [inline User_Authenticator.py]: Avoid hardcoding values , you can get input from user , From line 76 - 80 , there are many hardcoded values

**Review by Anish-M-code — CHANGES_REQUESTED** [review]

**Comment by Anish-M-code** [comment]
Also @bvks2020sameer avoid unnecessary blank lines , it feels odd.

**Comment by bvks2020sameer** [comment]
The above mentioned changes have been done

**Comment by Anish-M-code** [comment]
@bvks2020sameer thanks for contributing.

**Comment by bvks2020sameer** [comment]
Thankyou. Looking more to collaborate on . Happy Hacking

**Pull request merged**